### PR TITLE
frontend: remove store decorator from `<Spinner/>`.

### PR DESCRIPTION
**Pull request by shonsirsha**
To reduce ambiguity, we would like to manually pass `guideExists` prop to `<Spinner />` instead of relying totally on the `SharedProps` coming from the store (the `share` decorator).

The `guideExists` prop in the `<Spinner />` controls the render of the guide button when the `<Spinner />` component / view appears.

### CODE REVIEW

**Comment by thisconnect** [comment]
Looks like this will have a small conflict with staging-pocket. I suggest we either wait with merging this into master, or could merge into staging-pocket.

**Comment by shonsirsha** [comment]
@thisconnect let's merge this into staging-pocket.

**Comment by shonsirsha** [comment]
@thisconnect I was thinking we could merge this into staging-pocket, but let's just wait until staging-pocket gets merged to master.

**Review by thisconnect — APPROVED** [review]
untested LGTM

**Comment by thisconnect** [comment]
please rebase

**Pull request merged**
